packages/table/components/HeaderCell.tsx

```diff
@@ -2,6 +2,6 @@ import { default as Cell } from "./Cell";
 import styled from "react-emotion";
 
 export default styled(Cell)`
-  text-transform: uppercase;
+  text-transform: capitalize;
```

Comment:
Nice 👍

Reply:
It's marked as a bug, so I guess it should be a `fix`.